qemu/tests/cfg/blockdev_full_backup_nonexist_target.cfg

```diff
@@ -0,0 +1,24 @@
+- blockdev_full_backup_nonexist_target:
+    only Linux
+    virt_test_type = qemu
+    type = blockdev_full_backup_nonexist_target
+    kill_vm_on_error = yes
+    storage_pools = default
+    storage_type_default = "directory"
+    images += " src1"
+    start_vm = no
+    not_preprocess = yes
+    storage_pool = default
+    image_size_src1 = 200M
+    image_name_src1 = "src1"
+    force_create_image_src1 = yes
+    force_remove_image_src1 = yes
```

Comment:
use remove_image instead of force_remove_image, the latter one is not used for a regular image removal

Reply:
Have update, thanks zhenchao.